react/src/pages/addParticipants/DataEntryTable.tsx

```diff
@@ -60,19 +60,55 @@ const useTableStyles = makeStyles(theme => ({
     },
 }));
 
-const defaultOptionals = ["notes", "sex", "input_hpf_path"];
+const fallbackColumns = ["notes", "sex", "input_hpf_path"];
+
+function getEnvColumns(): Array<keyof DataEntryRowOptional> | null {
+    const envCols = process.env.REACT_APP_DEFAULT_OPTIONAL_COLUMNS;
+    console.log(envCols);
+    if (envCols !== undefined) {
+        const validFields = getDataEntryHeaders().optional;
+        const envFields = envCols.split(",");
+        const usableFields = envFields.filter(
+            field => !!validFields.find(valid => valid === field)
+        );
+        return usableFields as Array<keyof DataEntryRowOptional>;
+    } else {
+        return null;
+    }
+}
```

Comment:
I appreciate that we're verifying this configuration value. However, this code is running in the browser rather than at compile-time. We can't really afford to eject Create React App to add a webpack plugin that checks the environment variable. I think as a compromise we could leave this check to `NODE_ENV === "development"`. Also, returning an empty array here instead of `null` will probably make things either since there isn't a semantic difference between having a blank value and the absence of the envvar.